### ISSUE TITLE
Thousands of transient_timeout_timberloader records getting generated

### ISSUE THREAD

**Issue by nateluzod** [timber/timber#1128]
### Expected behavior

The `wp_options` table should not get bloated with transient records

### Actual behavior

`wp_options` currently has 12,000+ rows for `transient_timeout_timberloader`. I'm not sure if this is because of Timber or because of our server, but our DB grew from 5mb to 29mb over the past week and I'm just trying to identify the culprit. I'm not sure if this is the correct forum for this, apologies if not.

### Steps to reproduce behavior

Unsure how to reproduce since I'm not sure of the cause.

### What version of WordPress, PHP and Timber are you using?

We're running everything on a Pantheon package. 

* WordPress 4.5.2
* PHP 5.5
* Timber 0.22.6

### How did you install Timber? (for example, from GitHub, Composer/Packagist, WP.org?)
Timber was installed via WP-CLI.

**Comment by jarednova** [comment]
@nateluzod can you look into your PHP files and look for instance(s) of `TimberHelper::transient(...` or `Timber\Helper::transient(....` — I'm guessing that mixed with something else is causing the runaway

**Comment by rbvea** [comment]
@jarednova (@nateluzod's coworker here), there's no instance of `TimberHelper::transient` in our php files.  We're caching the timber templates using what's specified in the documentation:

```
$CACHE_TIME = array(10600, false); // don't cache for admin, cache for three hours otherwise
Timber::render('single.twig, $context, $CACHE_TIME);
```

Looks like the transient records aren't automatically expiring:

![screen shot 2016-08-01 at 10 14 54 am](https://cloud.githubusercontent.com/assets/949055/17307441/da80dc1c-57d0-11e6-8332-6e978b0f42f9.png)
```

**Comment by jarednova** [comment]
Thanks @rbvea — I'll check stuff out based on that!

**Comment by samuel-hauser** [comment]
same for our website, we got over 48'000 transient-entries in the 'wp-options' table and 'time to first byte' took up to ten seconds...

--> as soon as we stopped caching via timber 'time to first byte' was "normal" again.

**Comment by jarednova** [comment]
@samuel-hauser @nateluzod @rbvea I've looked into this a bit more and have a patch in #1190, let me know what you think!

**Comment by samuel-hauser** [comment]
Looks like a way to go. When or where will the patch go into the plugin? (Sorry, if I ask so noobish, have no idea about the workflow of such "things")

**Comment by jarednova** [comment]
@samuel-hauser — we do releases every two weeks. The next one is actually scheduled for tomorrow AM though!

**Comment by jarednova** [comment]
Closed by #1190

**Comment by nateluzod** [comment]
@jarednova we'll check it out. Thanks!
